portal_objects/workflows/BICseq2_add_confidence.yaml

```diff
@@ -0,0 +1,32 @@
+## Workflow information #####################################
+#     General information for the workflow
+#############################################################
+name: BICseq2_add_confidence
+description: Assign confidence classes to CNV calls from BIC-seq2.
+
+runner:
+  language: cwl
+  main: BICseq2_add_confidence.cwl
+
+software:
+  - granite@0.2.0
+
+## Input information ########################################
+#     Input files and parameters
+#############################################################
+input:
+
+  # File argument
+  input_vcf:
+    argument_type: file.vcf_gz
+
+## Output information #######################################
+#     Output files and quality controls
```

Comment:
there are no quality controls, is it still required to put this in the comment?

Reply:
same as above